Donation-platform/src/main/webapp/personalizationScript.js

```diff
@@ -91,6 +95,34 @@ function updatePersonalizedCardsOnPage(charities) {
     cards.innerHTML += toAdd;
 }
 
+/* Displays the instructions on how to order the tags */
+function showRankInstructions(tag) {
+    const tagRank = document.getElementById('sortable');
+    tagRank.innerHTML += '<label><b>2. Click and drag to order the causes:</b></label>';
+}
+
+/* Creates and displays the tag element in the rank given the tag's name */
+function placeSelectedTagInRank(tag) {
+    const tagRank = document.getElementById('sortable');
+
+    let tagElements = new Map([
```

Comment:
Could this be dynamically created? I can imagine if we have hundreds of tags we would not be able to hardcode every single one.

Reply:
The problem is that right now we don't store any sort of tag image/icon in the Tag class (correct me if I'm wrong). So regardless I'll need to manually create the icons associated with each tag. Once we coordinate tag images/icons and store them in the db I'll change this to be dynamic.

Reply:
We have an image source in the tag class. It'll be in my next merge.

Reply:
Ok, I am taking note to update this function once we do your merge.